src/Servers/IIS/IIS/src/Core/IISHttpContext.cs

```diff
@@ -198,7 +198,7 @@ protected void InitializeContext()
                 var pipe = new Pipe(
                     new PipeOptions(
                         _memoryPool,
-                        readerScheduler: PipeScheduler.ThreadPool,
+                        readerScheduler: PipeScheduler.Inline,
```

Comment:
nit: Is this worth putting a comment for posterity here to capture some of the reasoning why inline is important?

Reply:
Yes please

Reply:
Done.